databricks/koalas/frame.py

```diff
@@ -491,19 +493,20 @@ def _reduce_for_stat_function(self, sfun, name, axis=None, numeric_only=False):
                         # Pass in both the column and its data type if sfun accepts two args
                         col_sdf = sfun(col_sdf, col_type)
                     exprs.append(col_sdf.alias(name_like_string(label)))
+                    new_column_labels.append(label)
 
             sdf = self._sdf.select(*exprs)
-            pdf = sdf.toPandas()
-
-            if self._internal.column_labels_level > 1:
-                pdf.columns = pd.MultiIndex.from_tuples(self._internal.column_labels)
 
-            assert len(pdf) == 1, (sdf, pdf)
+            # The data is expected to be small so it's fine to transpose/use default index.
+            kdf = DataFrame(sdf)
+            internal = _InternalFrame(
+                kdf._internal.sdf,
+                index_map=kdf._internal.index_map,
+                column_labels=new_column_labels,
+                column_label_names=self._internal.column_label_names,
+            )
 
-            row = pdf.iloc[0]
-            row.name = None
-            # TODO: return Koalas series.
-            return row  # Return first row as a Series
+            return _col(DataFrame(internal).transpose())
```

Comment:
Should we use `with ks.option_context("compute.max_rows", None)` to avoid Spark job?